codecov_auth/views/bitbucket.py

```diff
@@ -63,8 +64,10 @@ def redirect_to_bitbucket_step(self, request):
             + b"|"
             + base64.b64encode(oauth_token_secret.encode())
         ).decode()
-        response.set_cookie(
-            "_oauth_request_token", data, domain=settings.COOKIES_DOMAIN
+        response.set_signed_cookie(
+            "_oauth_request_token",
+            encryptor.encode(data).decode(),
```

Comment:
so i am guessing `encryptor.encode()` encrypts and `.decode()` is to convert from bytes to str, right? awkward names

Reply:
Yeah that's right.  I agree it's confusing.  `encryptor.encrypt(data).decode()` would at least make a little more sense.